### ISSUE TITLE
Web support

### ISSUE THREAD

**Issue by karelklic** [mjablecnik/MyLogger#4]
Web support would be a fantastic addition to this package.

Currently, the initialization fails on [getApplicationSupportDirectory call in LogsDatabase](https://github.com/mjablecnik/MyLogger/blob/d1a047f5346853c2d7d1fba299d9fc723b42ca44/lib/services/logs_database.dart#L36C7-L36C7), which lacks web implementation.
Fortunately, sembast works fine on the web. It just needs to be [initialized differently](https://pub.dev/packages/sembast_web#usage).

What else needs an update?

AFAICT the core functionality (logging, filtering, retrieving logs) should work fine without any changes except the sembast initialization.

**Comment by mjablecnik** [comment]
Yes, it could be nice. I don't have a time to add web support and test it if works right (for mobile and web), but feel free to create PR and I will merge it ;)
